applications/FluidDynamicsApplication/tests/embedded_piston_test.py

```diff
@@ -105,7 +106,7 @@ def OutputSolutionStep(self):
             super(CustomFluidDynamicsAnalysis,self).OutputSolutionStep()
 
 
-@UnitTest.skipUnless(have_external_solvers,"Missing required application: ExternalSolversApplication")
+@UnitTest.skipUnless(have_external_solvers or have_mesh_moving,"Missing required application: ExternalSolversApplication or MeshMovingApplication")
```

Comment:
are you using the externalsolversapp?

Reply:
Yest, for the AMGCL (to make it faster).

Reply:
But the AMGCL is in the core ...

Reply:
@jcotela BTW I noticed that many of the fluid-tests still require the externalsolversapp bcs the amgcl was there initially and was moved at some point right?

Reply:
We can change them in a different PR.

Reply:
@philbucher I tried to remove as many unnecessary imports as I could, but quite a few fluid tests use SuperLU. We could consider migrating them, but I would do that on a separate PR, since I expect having to update a few result files.